sycl/source/detail/graph_impl.hpp

```diff
@@ -1195,6 +1241,10 @@ class exec_graph_impl {
       MPartitionsExecutionEvents;
   /// Storage for copies of nodes from the original modifiable graph.
   std::vector<std::shared_ptr<node_impl>> MNodeStorage;
+  /// If true, the L0 backend In-order CommandList optimization is enabled.
```

Comment:
Is this optimization L0 specific, or will be applied to all backends?

Reply:
Yes, in-order optimization is level-zero specific. That said, I've removed this member because we do not really need it (MEnableProfiling is enough).